requirements.txt

```diff
@@ -0,0 +1,5 @@
+pandas
```

Comment:
I think adding requirements.txt without any options or pinned versions does nothing different than using `install_requires`: https://github.com/santiment/sanpy/blob/master/setup.py#L20

Reply:
It is a bit noisy, so I've removed it.